docker/run_update_data.sh

```diff
@@ -6,18 +6,25 @@
 
 set -eo pipefail
 
+HOSTUSER=$(id -u):$(id -g)
+
+# The assumption is that you're running this from /app inside the container
+CACHEDIR=/app/.cache/ftpscraper
+
 # Fetch and update release information for these products (comma-delimited)
 PRODUCTS="firefox"
 CRONTABBER="docker-compose run crontabber python socorro/cron/crontabber_app.py"
 
 
-# Fetch release data
-${CRONTABBER} --job=ftpscraper \
-       --crontabber.class-FTPScraperCronApp.products=${PRODUCTS}
+# Fetch release data -- do it as the user so it can cache things
+docker-compose run -u "${HOSTUSER}" crontabber python socorro/cron/crontabber_app.py \
+               --job=ftpscraper \
+               --crontabber.class-FTPScraperCronApp.cachedir=${CACHEDIR} \
+               --crontabber.class-FTPScraperCronApp.products=${PRODUCTS}
```

Comment:
This has to run as the host user so that the file permissions are right. It's the same thing as what the `docker/as_me.sh` script does, but that specifically runs the processor container and we need to run the crontabber one.